hooks/basic/collector.py

```diff
@@ -263,6 +263,14 @@ def settings_schema(self):
                     "description": "Resolution order to follow when multiple item types"
                                    "are available the same extension, lower resolution order gets higher priority."
                                    "Item type with a matching work_path_template, gets priority of -1."
+                },
+                "ignore_sequences": {
+                    "type": "bool",
+                    "default_value": False,
+                    "allows_empty": True,
+                    "description": "Setting that ignores file sequences intentionally."
```

Comment:
"Setting this to True will force the collected items of this type to be treated as single frame publishes."